### ISSUE TITLE
add --trial_run

### ISSUE THREAD

**Issue by jgoeders** [byu-cpe/ygrader#28]

**Comment by jgoeders** [comment]
These have been added to `set_other_options`

```
   dry_run_first: bool
            Perform a dry run, calling your callback function to perform grading, but not updating the grades CSV file.
            The callback is only run for the first student.
        dry_run_all: bool
            Perform a dry run, calling your callback function to perform grading, but not updating the grades CSV file.
            The callback is run for each student.
```

**Comment by jgoeders** [comment]
See https://ygrader.readthedocs.io/en/latest/documentation.html#ygrader.grader.Grader.set_other_options

**Comment by jgoeders** [comment]
Fixed in 1.1.17

**Comment by jgoeders** [comment]
`pip install ygrader==1.1.17`
